### ISSUE TITLE
I define the tablename to `auth_users`, but it takes no effect.

### ISSUE THREAD

**Issue by laozhu** [gi0baro/weppy#118]
When I change the tablename to `blog_authors`, the tablename will change to it, but `auth_users` takes no effect, I want the same naming style with other auth tables. :(

```python
from weppy.tools.auth import AuthUser
from weppy.dal import has_many


class User(AuthUser):
    tablename = "auth_users"
    # tablename = "blog_authors"
    has_many("posts", "categories", "tags")
```

**Comment by gi0baro** [comment]
@laozhu Can you try this for me?

```python
from weppy.tools.auth import AuthUser as _AuthUser
from weppy.dal import has_many

class AuthUser(_AuthUser):
    has_many("posts", "categories", "tags")
```

**Comment by laozhu** [comment]
@gi0baro The tablename is `users` too.

**Comment by gi0baro** [comment]
@laozhu wait, if you rename your model `AuthUser` the tablename is **users**? This would be definitely strange.

**Comment by gi0baro** [comment]
Anyway, I will inspect this later today and publish a fix.

**Comment by laozhu** [comment]
@gi0baro I defined Model like this, Post / Category / Tag model belongs to user.

```python
from weppy.tools.auth import AuthUser as _AuthUser
from weppy.dal import Model, has_many, belongs_to


class User(_AuthUser):
    tablename = "auth_users"
    has_many("posts", "categories", "tags")


class Post(Model):
    tablename = "blog_posts"
    belongs_to("user")
    has_many("tags", "categories")
    ...


class Category(Model):
    tablename = "blog_categories"
    belongs_to("user")
    has_many("posts")
    ...


class Tag(Model):
    tablename = "blog_tags"
    belongs_to("user")
    has_many("posts")
    ...
```

**Comment by gi0baro** [comment]
@laozhu this is published in 0.6.4
